src/Cassandra/RowPopulators/Row.cs

```diff
@@ -133,9 +133,9 @@ public virtual bool IsNull(int index)
         /// </summary>
         public CqlColumn GetColumn(string name)
         {
-            return !ColumnIndexes.TryGetValue(name, out var index) 
-                ? null 
-                : Columns[index];
+            return !ColumnIndexes.TryGetValue(name, out var index)
+                ? Columns[index]
+                : null;
```

Comment:
```suggestion
            return ColumnIndexes.TryGetValue(name, out var index)
                ? Columns[index]
                : null;
```
this change is what is causing the test failures (at least some of them)

Reply:
I'm looking at this, I didn't expect to run the tests.

Reply:
You forgot to swap the lines when you removed the `!`, not a big deal. The build should return to green after this is fixed.

Reply:
@joao-r-reis thanks for the interaction, let me ask you a question.

Is there any limitation in the provider for ScyllaDB, is anything not yet supported?

Reply:
We support this driver for use with Apache Cassandra and any other backend that is fully compliant with Cassandra's native protocol and CQL specifications. 
We do not test this driver with ScyllaDB and do not monitor the ScyllaDB implementation for functionality that would impact this driver's capability with that database.

Reply:
I'm going to create another project focused on ScyllaDB based on this driver and start doing deeper tests with ScyllaDB, my main focus is ScyllaDB.
I will keep all copyrights, initially it will be a project focused on testing the Driver and compatibility.
I'm enjoying everything I saw here.
Congratulations to you and @datastax.